devtools/server/actors/replay/module.js

```diff
@@ -1181,17 +1181,6 @@ if (isRecordingOrReplaying) {
         onStreamData(value);
       }
 
-      // For some reason, request streams don't appear to be marked closed properly.
-      // Firefox doesn't support using an actual ReadableStream for fetch bodies,
-      // so every request body SHOULD have all of its data available immediately,
-      // meaning that we can immediately consider all request body streams to
-      // end once they have been read.
-      if (isRequestBody) {
-        onStreamEnd();
-        stream.close();
-        return;
-      }
```

Comment:
This appears to have been because the old tee was buggy and didn't close this stream properly. This is fixed by all those `mSink->Close()` calls.